composer.json

```diff
@@ -3,7 +3,7 @@
   "description": "Shopgate base module for Magento 2, handles communication with merchant API and routing calls",
   "license": "Apache-2.0",
   "type": "magento2-module",
-  "version": "2.9.16",
+  "version": "2.10.0",
```

Comment:
We are not really using semantical versioning yet as it is supposed to be. I would prefer version 2.9.17 for now.

Reply:
And you need to point to the development branch instead of the master. On release the develop is merged into master.